### ISSUE TITLE
sigOp counts not adapted to the segwit rules

### ISSUE THREAD

**Issue by jqriyv** [bitcoinj/bitcoinj#3081]
I found that the new SDK does not take into account the Witness case when calculating the sigops of a transaction
![image](https://github.com/bitcoinj/bitcoinj/assets/5435244/6bfaebcc-b9a4-42c4-b875-001a481c653c)

**Comment by schildbach** [comment]
Can you say what are you using the `getSigOpCount()` method for?

Afaicr, it's part of transaction validation.

**Comment by jqriyv** [comment]
I should display a tx sigops in explorer
![image](https://github.com/bitcoinj/bitcoinj/assets/5435244/85039368-36df-4cdd-b48f-a03a13e10da3)
tx hash:1cb6ab834ffc8f1a689aa25a072878723204d204269ca2d10cef787d44c70ce8
use bitecoinj get the value is 4

**Comment by schildbach** [comment]
Indeed, we didn't change how sigops are counted. The "new" rules are here: https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#sigops

In the above example, this is all non-segwit in- and outputs, so the sigop counts should be quadrubled.
